openchain-blockchain-coinprism/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Openchain validator (single-node)",
+  "description": "This template creates single-node Openchain validator node. The endpoint is exposed over HTTP so it is only recommended for testing and development purposes. Openchain is a Blockchain technology stack suited for organizations wishing to issue and manage digital assets in a robust, secure and scalable way. See openchain.org for more information.",
+  "summary": "This template creates a test Openchain validator node.",
+  "githubUsername": "flavien",
```

Comment:
@marleyg Since @flavien is not a MSFT employee, he should sign the CLA for submission